resim/testing/fuzz_helpers.hh

```diff
@@ -122,4 +122,39 @@ bool verify_equality(
     const google::protobuf::Timestamp &a,
     const google::protobuf::Timestamp &b);
 
+// Generate a random vector of T between size 3 and 7 assuming that
```

Comment:
[nit] some brief comment on the choice of limits would be helpful.

Reply:
Good call! I've added some comments here.